DER/DerivativesContracts/Swaps.rdf

```diff
@@ -239,6 +243,38 @@
 		<skos:definition>floating leg of a swap that depends on some statistical measure of the performance of the underlier</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-drc-swp;RateBasedLeg">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-swp;SwapLeg"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-rtd-rtd;RateBasedObservable"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-swp;isLegOf"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-drc-swp;RatesSwap"/>
```

Comment:
Odd to have the upward restriction, which we don't for other legs. Also the cardinality seems wrong - sure it should be exactly 1 (a given leg cannot be part of many swaps).